docs/releases/4.1.txt

```diff
@@ -267,6 +267,19 @@ Forms
   as the template implements ``<fieldset>`` and ``<legend>`` to group related
   inputs and is easier for screen reader users to navigate.
 
+  The div-based rendererring will become the default rendering style from Django
```

Comment:
renderererererering

```suggestion
  The div-based rendering will become the default rendering style from Django
```

Reply:
Channelling the inner Terry Pratchett...

> He knew how to start spelling it, but not how to stop.